app/editor/src/features/content/form/PropertiesContentForm.tsx

```diff
@@ -2,46 +2,94 @@ import { Button, ButtonVariant } from 'components/button';
 import { IOptionItem, OptionItem, RadioGroup, SelectDate } from 'components/form';
 import { FormikCheckbox, FormikDropdown, FormikText, FormikTextArea } from 'components/formik';
+import { Modal } from 'components/modal/Modal';
 import { Row } from 'components/row';
+import { Upload } from 'components/upload';
 import { useFormikContext } from 'formik';
+import { ITagModel, ITimeTrackingModel, IUserModel } from 'hooks';
+import useModal from 'hooks/modal/useModal';
 import React from 'react';
 import { useLookup } from 'store/hooks';
+import { useKeycloakWrapper } from 'tno-core';
+import { getSortableOptions } from 'utils';
 
 import { expireOptions, summaryOptions, toningOptions } from './constants';
 import { IContentForm } from './interfaces';
+import { TimeLogTable } from './TimeLogTable';
 
 export interface IContentSubForms {
   setContent: (content: IContentForm) => void;
   content: IContentForm;
 }
 
+/** The sub form of the ContentForm when the Properties Tab is selected. */
 export const PropertiesContentForm: React.FC<IContentSubForms> = ({ setContent, content }) => {
-  const [{ series: lSeries }] = useLookup();
-  const { values } = useFormikContext<IContentForm>();
+  const [{ series: lSeries, categories, tags, users }] = useLookup();
+  const { values, setFieldValue, handleChange } = useFormikContext<IContentForm>();
+  const [userTags, setUserTags] = React.useState<string[]>();
+  const [categoryTypes, setCategoryTypes] = React.useState<IOptionItem[]>([]);
+  const keycloak = useKeycloakWrapper();
+  const userId = users.find((u: IUserModel) => u.displayName === keycloak.getDisplayName())?.id;
 
+  React.useEffect(() => {
+    setCategoryTypes(getSortableOptions(categories));
+  }, [categories]);
+
+  const { isShowing, toggle } = useModal();
+
+  let timeLog = values.timeTrackings;
   const [series, setSeries] = React.useState<IOptionItem[]>();
 
   const formatTime = (date: string) => {
     const converted = new Date(date);
-    return !!converted.getTime()
-      ? `${converted.getHours()}:${converted.getMinutes()}:${converted.getSeconds()}`
-      : '';
+    return !!converted.getTime() ? converted.getTime() : '';
   };
 
   React.useEffect(() => {
     setSeries(lSeries.map((m: any) => new OptionItem(m.name, m.id)));
   }, [lSeries]);
 
+  const tagMatch = /(?<=\[).+?(?=\])/g;
+  let validTags;
+
+  const getTotalTime = () => {
+    let count = 0;
+    if (values.timeTrackings.length === 0) {
+      return 0;
+    }
+    values.timeTrackings.forEach((t: ITimeTrackingModel) => (count += Number(t.effort)));
+    return count;
+  };
+
+  const [effort, setEffort] = React.useState(getTotalTime());
+
+  /** capture user's tags when entering them in the summary field */
+  React.useEffect(() => {
+    var result;
+    if (values.summary) {
+      result = values.summary.match(tagMatch)?.toString();
+      setUserTags(result?.split(', '));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This one worries me and probably has issues.

Reply:
Trying to figure out how to work around it, it wants me to include the regex statement which sends it into a loop